packages/sit-onyx/src/components/OnyxSwitch/OnyxSwitch.stories.ts

```diff
@@ -0,0 +1,48 @@
+import { defineStorybookActionsAndVModels } from "@sit-onyx/storybook-utils";
+import type { Meta, StoryObj } from "@storybook/vue3";
+import OnyxSwitch from "./OnyxSwitch.vue";
+
+/**
+ * Toggle switches are a common ui element used to controll binary states, such as on/off, enable/disable/, or active/inactive.
+ * They consist of a toggle mechanism that allow users to switch between two distinct states with a simple interaction.
+ */
+const meta: Meta<typeof OnyxSwitch> = {
+  title: "components/OnyxSwitch",
+  ...defineStorybookActionsAndVModels({
+    component: OnyxSwitch,
+    events: ["update:modelValue"],
+  }),
+};
+
+export default meta;
+type Story = StoryObj<typeof OnyxSwitch>;
+
+/**
+ * This example shows the default checked state of the switch.
+ */
+export const DefaultChecked = {
+  args: {
+    label: "Switch",
+    modelValue: true,
+  },
+} satisfies Story;
+
+/**
+ * This example shows the default unchecked state of the switch.
+ */
+export const DefaultUnchecked = {
+  args: {
+    label: "Switch",
+    modelValue: false,
+  },
+} satisfies Story;
+
+/**
+ * This example shows the invalid state of the switch.
+ */
+export const Invalid = {
+  args: {
+    ...DefaultChecked.args,
+    errorMessage: "Error message",
+  },
+} satisfies Story;
```

Comment:
```suggestion
/**
 * This example shows the default switch.
 */
export const Default = {
  args: {
    label: "Switch",
  },
} satisfies Story;

/**
 * This example shows the switch that is initially checked.
 */
export const Checked = {
  args: {
    label: "Switch",
    modelValue: true,
  },
} satisfies Story;

/**
 * This example shows the invalid state of the switch.
 */
export const Invalid = {
  args: {
    ...Checked.args,
    errorMessage: "Error message",
  },
} satisfies Story;
```

I think its more intuitive if the switch is unchecked in the default example